data/local_services.csv

```diff
@@ -125,3 +125,4 @@ LGSL,Description,Providing Tier
 1742,Find the location of weighbridges,county/unitary
 1743,Find out about the Care Act 2014,county/unitary
 1788,"Find out about support for children who have difficulties with speech, language or communication",county/unitary
+1826,"The Test and Trace Support Payment is for people on a low income, who must legally self-isolate for 14 days. either because they have tested positive for coronavirus or have recently been in close contact with someone who has tested positive and cannot work from home during this period. The scheme is administered by local authorities",all
```

Comment:
The description seems a bit long here - do we know where it gets used and is it suitable?

Reply:
Hi @theseanything thanks for the comment - I'm not 100% sure where, or even if, it's used to be honest.  I just C&P'd from here https://standards.esd.org.uk/?uri=service%2F1826 - which looks like what everyone has done for the `enable...` PR's

Reply:
Having said that - I am (perhaps like yourself) a little suspicious of the `Find <out>...` phraseology.  I'll do some digging to see if I can find out if this is significant.

Reply:
Ok, so I'm wrong...

- Line 127 (1788) has a description of [`Work with children who have difficulties with speech, language or communication to improve their behaviour and ability to socialise`](https://standards.esd.org.uk/?uri=service%2F1788) but uses [`Find out about support for children who have difficulties with speech, language or communication`](https://www.gov.uk/learning-activities-for-your-child)

- Line 125 (1742) has a description of [`Public weighbridges are available for use by businesses or the general public for the weighing of vehicles and objects where determination of a specific weight/load is important, including ensuring that vehicles are not overloaded. Public weighbridges may be provided by the council or they may provide information on privately operated weighbridges in the area.`](https://standards.esd.org.uk/?uri=service%2F1742) and uses [`Find your nearest weighbridge if you need to weigh your van, lorry, trailer, tractor or other vehicle.`](https://www.gov.uk/find-weighbridge)

So, I'm unsure if these have any use or meaning.

Reply:
Have changed the description to be shorter and closer to the other examples.